src/common/components/ReviewItem/ReviewItem.tsx

```diff
@@ -1,25 +1,23 @@
 import { reviewItemTheme, type ReviewItemVariants } from "./ReviewItem.theme";
-import { ReviewBody } from "./ReviewBody";
-import { ProfileSchool } from "./ProfileSchool";
-import { ProfileReviewer } from "./ProfileReviewer";
-import { getHumanReadableTimestampDelta } from "@/common/functions";
-import { ThumbUpFilledIcon } from "@/common/components/CustomIcon";
 import { LockCtaOverlay } from "@/common/components/LockCtaOverlay";
-import { Button } from "@/common/components/Button";
-
+import ReviewHeader from "@/common/components/ReviewItem/ReviewHeader";
+import ReviewModal from "@/common/components/ReviewModal/ReviewModal";
 // TODO: to replace with prisma generated types
 export type ReviewLabel = {
   name: string;
-  typeof: "professor" | "course";
 };
 
 export type Review = {
+  id: string;
   body: string;
   courseCode: string;
   username: string;
   likeCount: number;
   labels: ReviewLabel[];
   createdAt: number;
+  reviewedUniversityId: number;
+  reviewFor: "professor" | "course";
+  professorName?: string;
 };

```

Comment:
thought: this `Review` seeems to be well shared across all review related component, we can probably think about moving this to `@/common/module/review/types.ts`

Reply:
good point, done https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/79fb3d79570fd8a8455ef4e70dcc687af5bbf8b4